terminal/termbox/cell_options.go

```diff
@@ -60,6 +60,13 @@ func cellOptsToFg(opts *cell.Options) (tbx.Attribute, error) {
 	if opts.Strikethrough {
 		return 0, errors.New("Termbox: Unsupported attribute: Strikethrough")
 	}
+	if opts.Inverse {
+		a |= tbx.AttrReverse
+	}
+	// FIXME: Termbox doesn't have a blink attribute
```

Comment:
(nit, optional) We might be able to remove the FIXME comments now, since there probably isn't anything left to fix. I.e. we don't really plan to add support for this in `termbox-go`.